etc/generate-docs.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# usage: ./etc/generate-docs.sh [new version string]
+
+# exit if any command fails
+set -e
+
+if ! command -v jazzy > /dev/null; then
+  gem install jazzy || { echo "ERROR: Failed to locate or install jazzy; please install yourself with 'gem install jazzy' (you may need to use sudo)"; exit 1; }
+fi
+
+version=${1}
+
+jazzy_args=(--clean
+            --author 'Matt Broadstone, Kaitlin Mahar, and Patrick Freed' 
+            --readme "docs/README.md" 
+            --author_url https://github.com/mongodb/mongo-swift-driver 
+            --github_url https://github.com/mongodb/mongo-swift-driver 
+            --theme fullwidth 
+            --documentation "Guides/*.md" 
+            --github-file-prefix https://github.com/mongodb/mongo-swift-driver/tree/v${version} 
```

Comment:
this makes it so, if someone clicks one of the "view on github" links in the docs, it will take them to the place the code was at the time of the tag, rather than on master.